lib/linguist/languages.yml

```diff
@@ -1694,6 +1694,18 @@ Julia:
   color: "#a270ba"
   ace_mode: julia
 
+ Jupyter Notebook:
```

Comment:
There is an extra space at the beginning of this line. It might be what's causing the Travis build to fail.